plugin-annotation/src/main/java/com/mapbox/maps/plugin/annotation/generated/PointAnnotation.kt

```diff
@@ -65,8 +65,9 @@ class PointAnnotation(
     set(value) {
       value?.let {
         field = it
-        if (iconImage == null) {
-          iconImage = ICON_DEFAULT_NAME_PREFIX + it.generationId
+        if (iconImage == null || iconImage!!.startsWith(ICON_DEFAULT_NAME_PREFIX)) {
+          // User does not set iconImage, update iconImage to this new bitmap
+          iconImage = ICON_DEFAULT_NAME_PREFIX + it.hashCode()
```

Comment:
Shouldn't we revisit how `generationId` is generated instead of not using it here and introducing standard hash function?

Reply:
From the official docs: `generationId` changes whenever the bitmap is modified. This can be used as an efficient way to check if a bitmap has changed. Here we want to distinguish different bitmap, so use `hashCode` would be appropriate.